ChaosMod/Effects/db/Misc/MiscStuffGuns.cpp

```diff
@@ -0,0 +1,154 @@
+#include <stdafx.h>
+
+static Vector3 DegToRadian(const Vector3& angles)
+{
```

Comment:
This should already be available, right?
Please extract this (and the other duplicate) methods from this class and move them i.e. to the types.h

Reply:
the types.h in the scripthook sdk?

Reply:
Huh... you're right... Putting everything into the Vector3-Struct made it possible to use all those static helper function i provided (Lenght, DistanceTo, getDirectionForRotation and so on 🤔 
I would put it on there with all the other helper methods, but i don't know if we should move it into the ChaosMod instead :D 
@pongo1231 what do you think? Can we leave those helper methods in the types.h? Or should we move it into another helper header in the mod but "lose" the static funcs to general helper funcs? :)

Reply:
I don't have an issue with it being located in the types.h (for now) since it's highly unlikely for us to update any of the shv headers anyways (apart from natives.h).